src/app/components/overview-history-summary-header/overview-history-summary-header.component.html

```diff
@@ -0,0 +1,136 @@
+<div class="overview-summary-background">
+    <div
+        class="d-flex flex-column flex-md-row justify-content-between pb-3 position-relative"
+        *ngIf="hasResultQuantity"
+    >
+        <h2 class="box-title align-items-center">
+            {{ searchResultQuantity(tableSource) }} {{ resultUnitText }}
+        </h2>
+    </div>
```

Comment:
This entire block and referred inputs seems to be an unneded copy-paste from repo-list component